INSTALL.WIN.md

```diff
@@ -0,0 +1,100 @@
+# Windows
+
+## Install Natively
+
+### [Install Chocolatey](https://chocolatey.org/install#installing-chocolatey) (optional)
+
+### [Install git](https://git-scm.com/)
+[Download](https://git-scm.com/download/)
+or
+```bat
+choco install git
+```
+
+### [Install Ruby](https://www.ruby-lang.org/)
+[Download](https://rubyinstaller.org/downloads/)
+or
+```bat
+choco install ruby
+```
+
+### [Install nim](https://nim-lang.org/)
+1. [Download](https://nim-lang.org/install_windows.html)
+2. Add nim to PATH:
+```%USERPROFILE%\.nimble\bin```
+or 
+where your nim is installed (you can check it when installs).
+
+### [Install neverwinter_utils.nim](https://github.com/niv/neverwinter.nim)
+```bat
+nimble install neverwinter
+```
+
+### [Install nwnsc](https://gitlab.com/glorwinger/nwnsc)
+1. [Download](https://neverwintervault.org/project/nwnee/other/tool/nwnsc-nwn-enhanced-edition-script-compiler)
+2. Extract in some folder under your watch like ```%USERPROFILE%\bin```
+3. Add nwnsc.exe folder to PATH (optional)
+
+### [Install nwn-devbase](https://github.com/jakkn/nwn-devbase)
```

Comment:
Recursive documentation link.

Also, it should probably be said where nwn-devbase should be located on the system. Perhaps lead with

```suggestion
### Install nwn-devbase
Navigate to where you want the nwn-devbase sources located on your system. `%USERPROFILE%\src\` is a good alternative.
```

Reply:
Done.